packages/reports/src/Components/Elements/Modals/ServicesErrorModal/constants.ts

```diff
@@ -0,0 +1,16 @@
+import { localize } from '@deriv/translations';
+
+export const getTitle = (type?: string) => {
+    switch (type) {
+        case 'buy':
+            return localize('Purchase Error');
+        case 'cancel':
+            return localize('Deal Cancellation Error');
+        case 'contract_update':
+            return localize('Contract Update Error');
+        case 'sell':
+            return localize('Sell Error');
+        default:
+            return 'Error';
```

Comment:
Shouldnt this be localized also

Reply:
I'm not sure but I copied this component from dtrader package and it's not localized for some reason. I don't think we should make any changes to this existing logic.